torch/csrc/jit/frontend/schema_type_parser.cpp

```diff
@@ -206,7 +206,7 @@ TypePtr SchemaTypeParser::parseRefinedTensor() {
   std::vector<int64_t> strides;
   parseList(TK_NOTHING, ',', ')', [&] {
     // Extra handling for options like 'device' and 'requires_grad'
-    if (L.cur().kind == TK_IDENT) {
+    if (L.cur().kind == TK_IDENT && L.cur().text() != "SS") {
```

Comment:
Nit: There is a comment in the beginning showing what parser supports, might be worth mentioning symbolic shapes in examples as well.